install.sh

```
@@ -1520,7 +1649,8 @@ configure_web_port() {
     print_message "\n🔌 Checking web interface port availability..." "$YELLOW"
     
     if ! check_port_availability $WEB_PORT; then
-        print_message "❌ Port $WEB_PORT is already in use" "$RED"
+        local process_info=$(get_port_process_info "$WEB_PORT")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)